devops/extend_ebs_volume_testnet/main.py

```diff
@@ -0,0 +1,111 @@
+import os
+import sys
+import boto3
+import botocore
+from botocore.exceptions import ClientError
+
+EBS_size = 40
+BUCKET_NAME = 'harmony-benchmark'
+region_array = ['us-east-1', 'us-west-2']
+
+path_file_s3 = 'logs/2019/09/11/005018/'
+remote_dist_file = path_file_s3 + 'distribution_config.txt'
+
+home_dir = os.getcwd()
+temp_dir = home_dir + '/temp'
+try:
+    os.mkdir(temp_dir)
+except OSError:
+    print("Director exists, or creation of this directory %s failed" % temp_dir )
+else:
+    print("Successfully created this directory %s " % temp_dir)
+
+local_dist_file = temp_dir + '/distribution_config.txt'
+
+dict_ip_per_region = {}
+dict_ebs_id_region = {}
+
+session = boto3.Session(profile_name='default')
+
+def download_file_s3(bucketname, remote_file, local_file):
+    s3_client = boto3.resource('s3')
+    s3_client.Bucket(bucketname).download_file(remote_file, local_file)
+
+def create_ip_for_each_region(fpath):
```

Comment:
use of this function for what? no explanation.

Reply:
is it just parsing for distribution_config file?  is there any other code can be reuse?